spec/v3/extensions/belongs_to_spec.rb

```diff
@@ -12,7 +12,7 @@
     after      { repo.destroy; user.destroy }
 
     example { expect(repo.owner).to be_a(Travis::API::V3::Models::User) }
-    example { expect(::Repository.find(repo.id).owner).to be_a(::User)  }
+    example { expect(Travis::API::V3::Models::Repository.find(repo.id).owner).to be_a(Travis::API::V3::Models::User)  }
```

Comment:
Can you revert this change? This test is to make sure the patch doesn't affect the travis-core models.